docker/epicsBase/Dockerfile

```diff
@@ -1,13 +1,13 @@
 
-FROM python:3.12.1 as epicsbase
+FROM python:3.12.2 as epicsbase
```

Comment:
The changelog says, we updated to "Python 3.12.1". However, this looks like an update to 3.12.2. Can we update the changelog, please?